shared/util/saga.js

```diff
@@ -127,7 +127,7 @@ function safeTakeEveryPurePromise<A, RA>(
     const state: TypedState = yield select()
     const toPut = yield call(f, state, action)
     if (toPut) {
-      yield put(toPut)
+      yield Array.isArray(toPut) ? sequentially(toPut.map(p => put(p))) : put(toPut)
```

Comment:
i've changed how this works, lets discuss

Reply:
actually i changed simple, not promise really ,but still it doing a single put was on purpose